src/main/java/edu/kis/powp/jobs2d/drivers/adapter/LineDrawerAdapter.java

```diff
@@ -0,0 +1,36 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class LineDrawerAdapter implements Job2dDriver {
+	private int startX = 0, startY = 0;
+	private final ILine line = LineFactory.getSpecialLine();
```

Comment:
Hardcoded line type. What about 3.3.2 test?
Direct coupling with LineFactory class is unnecessary.